x-pack/libbeat/autodiscover/providers/aws/elb/_meta/fields.yml

```diff
@@ -0,0 +1,84 @@
+- key: elb_listener
+  title: "ELB Listener"
+  description: >
+    ELB Listeners
+  short_config: false
+  release: experimental
+  fields:
+    - name: elb_listener
+      type: group
+      description: >
+          jolokia contains metrics exposed via jolokia agent
+      fields:
+        - name: ip_address_type
+          type: keyword
+          level: core
+          required: true
+          example: "ipv4"
+          description: Whether this is an ipv4 or dual-stack IP
+        - name: state
+          type: keyword
+          level: core
+          example: active
+          description: Current state of the ELB
+        - name: type
+          type: keyword
+          level: core
+          example: application
+          description: Whether this is an application or network loadbalancer
+        - name: protocol
+          type: keyword
+          level: core
+          example: HTTP
+          description: The application layer protocol
+        - name: vpc_id
+          type: keyword
+          level: core
+          example: vpc-123456
+          description: ID of the VPC
+        - name: ssl_policy
+          type: keyword
+          level: core
+          description: The specific SSL policy
+        - name: load_balancer_arn
+          type: keyword
+          level: core
+          example: arn:aws:elasticloadbalancing:us-east-1:331574139922:loadbalancer/app/testytesty/a14c9934d2a6e438
+          description: ARN for the ELB itself
+        - name: security_groups
+          type: keyword
+          level: core
+          example: sg-a1b555d4
+          description: List of security groups for this
+        - name: created
+          type: date
+          level: core
+          example: 2019-06-12T21:55:14.490Z
+          description: Date of creation
+        - name: availability_zones
+          type: keyword
+          level: core
+          example: "us-east-1e"
+          description: List of AZs this is active in
+        - name: host
```

Comment:
In the docs you listed `host` on the top level but here it is nested. Same for port, is this correct?

Reply:
Yes, different host variable. The top level one isn't in the event, it's only available during config templating. That's a standard thing for autodiscover providers.

It's debatable whether we need to duplicate host here since it's part of `url.domain`, however it's part of the API response, and it feels weird leaving it incomplete.